Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,93 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            Console.WriteLine(message);
+            var i = Console.ReadLine();
+            var isNumeric = int.TryParse(i, out int number);
+            if (String.IsNullOrEmpty(i))
```

Comment:
I would recommend separating the if statement of the value that you are checking with a blank line form the rest of code.